google/cloud/storage/internal/signed_url_requests.cc

```diff
@@ -235,13 +235,14 @@ Status V4SignUrlRequest::Validate() {
 }
 
 std::string V4SignUrlRequest::Hostname() {
+  auto host = common_request_.host();
```

Comment:
Please change the unit tests to validate the new functionality.

Reply:
Added new unit tests